addOns/authhelper/src/main/zapHomeFiles/reports/auth-report-json/report.json

```diff
@@ -0,0 +1,27 @@
+{
+	"@programName": [[${programName}]],
+	"@version": [[${zapVersion}]],
```

Comment:
I don't know if we expose this. But would it make sense to also include authhelper and reports version(s)?

Reply:
I'm not so keen on this idea. I can see the user wanting to know which version of ZAP they used, but the add-on versions less so. And reports are for the user, not us. If we need support info from them then can ask for it.